src/main/resources/assets/lambdynlights/lang/de_de.json

```
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
   "lambdynlights.option.block_entities": "Blockentitäten",
```

```suggestion
  "lambdynlights.option.block_entities": "Blockobjekte",
```
This is the German term used in the Minecraft wiki, for consistency: https://minecraft-de.gamepedia.com/Blockobjektdaten